res/META-INF/plugin.xml

```diff
@@ -54,12 +56,28 @@
 		<stubIndex implementation="org.ice1000.julia.lang.psi.JuliaAbstractTypeDeclarationIndex"/>
 		<stubIndex implementation="org.ice1000.julia.lang.psi.JuliaTypeDeclarationIndex"/>
 		<stubIndex implementation="org.ice1000.julia.lang.psi.JuliaModuleDeclarationIndex"/>
+		<localInspection displayName="LSP"
```

Comment:
Can the display name be changed to "language server" or "julia language server"?